README.md

```diff
@@ -84,7 +84,12 @@ Tests are an important part of the development and evolution of this project,
 and new contributions are expected to include tests for any functionality
 change.  To run the tests, pass the `test` verb to the `bootstrap` script:
 
-    ./Utilities/bootstrap --build-tests test
```

Comment:
Can we keep the old version that doesn't require any parameter in the readme? By making tools simpler we can lover entrance barrier for contributors